runtime/vm/jvminit.c

```diff
@@ -1816,19 +1816,13 @@ IDATA VMInitStages(J9JavaVM *vm, IDATA stage, void* reserved) {
 			argIndex2 = FIND_ARG_IN_VMARGS(EXACT_MATCH, VMOPT_XXTRANSPARENT_HUGEPAGE, NULL);
 			{
 				/* Last instance of +/- TransparentHugepage found on the command line wins
-				 * Default to -XX:-TransparentHugepage for performance reasons
+				 *
+				 * Default to use OMR setting (Enable for all Linux with THP set to madvise)
 				 */
 				if (argIndex2 > argIndex) {
 					j9port_control(J9PORT_CTLDATA_VMEM_ADVISE_HUGEPAGE, 1);
 				} else if (argIndex > argIndex2){
```

Comment:
```suggestion
				} else if (argIndex > argIndex2) {
```

Reply:
This is a minor formatting nitpick in existing code.  I'm ambivalent to wether it's applied or not

Reply:
Fixed